documentation/asciidoc/topics/proc_enabling_grafana_dashboards.adoc

```diff
@@ -0,0 +1,38 @@
+[id='enabling-grafana-dashboards_{context}']
+= Creating {brandname} Dashboards
+{ispn_operator} provides global configuration properties that let you configure Grafana dashboards for {brandname} clusters.
+
+.Procedure
+
+. Create a `ConfigMap` named `infinispan-operator-config`.
++
+[source,options="nowrap",subs=attributes+]
+----
+include::yaml/infinispan_operator_config.yaml[]
+----
++
+. Specify a namespace in the `grafana.dashboard.namespace` property and {ispn_operator} creates a Grafana dashboard for your {brandname} cluster in that namespace.
++
+[NOTE]
+====
+The namespace you specify must be one that {ispn_operator} watches.
+====
++
+You can modify this property while {ispn_operator} is running.
+Deleting the value of this property deletes the dashboard.
++
+. Specify values for other `data.grafana.dashboard` fields as required.
+. Add `infinispan-operator-config` to the {ispn_operator} namespace.
++
+[source,options="nowrap",subs=attributes+]
+----
+$ oc apply -f infinispan-operator-config.yaml
+----
++
+. Delete the {ispn_operator} pod to restart it.
+
+.Verification
+
+. Open the {osweb} and navigate to the Grafana Operator.
+. Select the **Grafana Dashboard** tab and check that the dashboard object is created.
+. Open the Grafana UI and search for the {brandname} dashboard.
```

Comment:
UI will be located at
`oc get routes grafana-route -o jsonpath=https://"{.spec.host}"`